src/core/client/tool-confirmation/allowed-tools-provider/factory.ts

```
@@ -1,17 +1,25 @@
-import { AllowedToolsProvider } from './types.js';
 import { InMemoryAllowedToolsProvider } from './in-memory.js';
+import type { IAllowedToolsProvider as IAllowedToolsProvider } from './types.js';
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)